tests/docker/languages/Dockerfile.testing.rust.big_endian.1_30_1

```
@@ -0,0 +1,15 @@
+FROM rust:1.30.1-slim-stretch as base
+
+WORKDIR /setup
+RUN apt -qq update >/dev/null
+RUN apt -qq install -y curl gcc-mips-linux-gnu
+RUN curl https://sh.rustup.rs > rustup.sh
+RUN cat rustup.sh | sh -s -- -y --default-toolchain none
+RUN rustup target add mips-unknown-linux-gnu
+
+
+WORKDIR /code
+ADD . .
+WORKDIR /code/tests/rust_usage_test
+ENV CARGO_TARGET_MIPS_UNKNOWN_LINUX_GNU_LINKER=gcc-mips-linux-gnu
```

So, while the Debian package name for the MIPS GCC toolchain is `gcc-mips-linux-gnu`, the actual executable that gets installed is called `mips-linux-gnu-gcc`, and it should be located at `/usr/bin/mips-linux-gnu-gcc` so it'll be in your path.

```suggestion
ENV CARGO_TARGET_MIPS_UNKNOWN_LINUX_GNU_LINKER=mips-linux-gnu-gcc
```